xoa/data_source_hosts.go

```diff
@@ -46,9 +47,14 @@ func dataSourceHostsRead(d *schema.ResourceData, m interface{}) error {
 	if err != nil {
 		return err
 	}
-	hosts, err := c.GetSortedHosts(client.Host{Pool: pool[0].Id, Tags: tags}, d.Get("sort_by").(string), d.Get("sort_order").(string))
+	searchHost := client.Host{
+		Pool:   pool[0].Id,
+		Tags:   tags,
+		Cpus:   client.CpuInfo{},
+		Memory: client.HostMemoryObject{}}
```

Comment:
Can you explain this change? The former code should have been initialized in the same way behind the scenes.